index.js

```diff
@@ -6,15 +7,258 @@ function ask(questionText) {
+const messHall = new Room('Mess Hall', 'You are now in the mess hall.\n\nIt appears that the last meal listed on the menu board was Taco Suprise.\nYou are not feeling particularly hungry but thirst is gnawing at you...\nMaybe this opened crate of "energyDrink" is still good?\n\n ', ['energyDrink'], false, true)
+const generatorRoom = new Room('Generator Room', 'You are now in the generator room.\n\nAs you move through the archway into this room, you notice various consoles full of buttons and gauges, shelves with parts bins stacked up, as well as a pile of spare parts, including a fuse, laid out upon a small workbench.\n\n ', ['fuse'], false, true)
+
+//item class with descriptions
+class Item {
+  constructor(name, description, takeable) {
+    this.name = name,
+      this.desc = description,
+      this.takeable = takeable
+  }
+
+}
+const energyDrink = new Item('Energy Drink', 'That energy drink sure looks refreshing.', true)
+const coveralls = new Item('coveralls', 'Coveralls are worn by the enlisted staff.', false)
+const ECM = new Item('ECM', 'The ECM is necessary to switch fuels for a quantum jump.', true)
+const coffeeCup = new Item('Coffee Cup', 'Caffeine makes the days fly by.', false)
+const journal = new Item('Journal', 'This is a daily record kept by officers.', false)
+const notepad = new Item('notepad', '\n"Boss keeps telling me I\'m a "liability" to security...\n So I should probably remember this passcode: 1223.\n I\'ll show the bossman this time."\n\n', true)
+const blueprint = new Item('Blueprint', 'At the bottom of the blueprint, it reads: thanks for all the fish.', true)
+const key = new Item('Officer Quarters key', 'This key grants access to the Officer Quarters.', true)
+const fuse = new Item('Fuse', "generic fuse, probably useful somehow", true)
+// -------------------------------- transition rules ------------------------------------
+//allowable room transitions
+let shipRooms = {
+  hangar: { canChangeTo: ['engineBlock', 'hangarControlRoom', 'messHall'] },
+  hangarControlRoom: { canChangeTo: ['hangar', 'officerQuarters'] },
+  engineBlock: { canChangeTo: ['hangar', 'messHall'] },
+  officerQuarters: { canChangeTo: ['hangarControlRoom', 'messHall'] },
+  messHall: { canChangeTo: ['generatorRoom', 'officerQuarters', 'engineBlock', 'hangar'] },
+  generatorRoom: { canChangeTo: ['messHall'] }
+}
+
+//--------------------------------- Lookup Tables --------------------------
+
+//Lookup table for allowable movement or action by the player
+let playerMovements = {
+  move: ['west', 'east', 'north', 'south'],
+  drink: ['take a drink', 'drink'],
+  take: ['take'],
+  leave: ['leave', 'drop'],
+}
```

Comment:
This doesn't get used ever, you could either incorporate it the way you did for the roomLookUp tables, or remove it to cut down on unused code.